### PR TITLE
[Ldap] Update ldap.rst

### PR DESCRIPTION
**Pull request by jraller**
include documentation of the filter option for the query command.

### CODE REVIEW

File: components/ldap.rst [hunk 1]
```diff
@@ -115,6 +115,11 @@ to the ``LDAP_SCOPE_BASE`` scope of :phpfunction:`ldap_read`) and ``SCOPE_ONE``
 
     $query = $ldap->query('dc=symfony,dc=com', '...', ['scope' => QueryInterface::SCOPE_ONE]);
 
+To retrieve only specific attributes you can specify which attributes you want returned using 
+the ``filter`` option.
```
Comment (OskarStark) [inline components/ldap.rst]:
```suggestion
the ``filter`` option::
```

**Comment by OskarStark** [comment]
Is this code already available in Symfony 4.4?

**Comment by jraller** [comment]
> Is this code already available in Symfony 4.4?

Using blame to look at /Adapter/ExtLdap/Query.php in the callSearchFunction line shows that there was a refactor on 2020-10-07, but that the code that was refactored had `$this->options['filter']` already at that point.

I'm seeing this as far back as 3 years ago, so I'm feeling like it was available in 4.4.

**Comment by OskarStark** [comment]
Thanks for your feedback ✅

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Jason, thanks a lot for adding these missing docs and for taking the time to investigate where should we merge this (we merged it in 4.4 and up). Also, congrats on your first Symfony Docs contribution 🎉

**Pull request merged**